### PR TITLE
ACI Phase II

### PR DESCRIPTION
**Pull request by JosepBove**
https://governance.aave.com/t/arfc-aci-phase-ii/15138/19

### CODE REVIEW

File: src/20231029_AaveV3Ethereum_ACIPhaseII/ACIPhaseII.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: "ACI Phase II"
+author: "@émarczeller + Alice Rozengarden (@Rozengarden - Aave-chan initiative)"
```
Comment (Zer0dot) [inline src/20231029_AaveV3Ethereum_ACIPhaseII/ACIPhaseII.md:3]:
```suggestion
author: "@marczeller + Alice Rozengarden (@Rozengarden - Aave-chan initiative)"
```

**Comment by JosepBove** [comment]
Comments applied!

**Comment by sendra** [comment]
A pull from main, and forge install is needed, as aave-helpers was updated

File: src/20231029_AaveV3Ethereum_ACIPhaseII/AaveV3Ethereum_ACIPhaseII_20231029.t.sol [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      assertEq(depositGHO, ACTUAL_STREAM_AMOUNT_GHO);
+      assertEq(tokenAddressGHO, AaveV3EthereumAssets.GHO_UNDERLYING);
+      assertEq(stopTimeGHO - startTimeGHO, STREAM_DURATION);
+      assertEq(remainingBalanceGHO, ACTUAL_STREAM_AMOUNT_GHO);
+    }
+
+    // checking if the ACI can withdraw from the stream
+
+    vm.startPrank(ACI_TREASURY);
+    vm.warp(block.timestamp + STREAM_DURATION + 1 days);
```
Comment (Zer0dot) [inline src/20231029_AaveV3Ethereum_ACIPhaseII/AaveV3Ethereum_ACIPhaseII_20231029.t.sol:70]: In the future, I'd recommend testing withdrawing prior to the end of the stream for completeness. No need to affect this now, as the streams are out of scope here, and the test does ensure it's created. 

Just something to keep in mind for future stream proposals.

Comment (marczeller) [reply src/20231029_AaveV3Ethereum_ACIPhaseII/AaveV3Ethereum_ACIPhaseII_20231029.t.sol:70]: noted for future proposals.

**Review by rustboyar — APPROVED** [review]

**Review by Zer0dot — APPROVED** [review]

**Pull request merged**